### ISSUE TITLE
elastic-package build command copies hidden files/directories into build directory

### ISSUE THREAD

**Issue by mrodm** [elastic/elastic-package#1156]
Relates https://github.com/elastic/elastic-package/issues/1154

If a package is defined in a separated repository outside integrations repository, hidden files or directories like `.idea`, `.gitignore`, etc... are copied into the build target directory.

These files and folders should also be skipped.

**Comment by ruflin** [comment]
As inspiration, seems like Docker faced similar issues and came up with the Dockerignore file: https://docs.docker.com/engine/reference/builder/#dockerignore-file

More general, I think there are 2 approaches: Specify what should be added or specify what should be ignored. We currently follow the ignore approach. But I wonder if this is needed as the package-spec knows pretty well which files should be in the package (and are useful) and the rest can be dropped.

**Comment by jsoriano** [comment]
> the package-spec knows pretty well which files should be in the package (and are useful) and the rest can be dropped.

Yes, the package-spec itself could expose the list of files to package.
